### ISSUE TITLE
API should have versioning in URL

### ISSUE THREAD

**Issue by liferealized** [bitpay/insight-api#296]
It would be good to have versioning within the API URL like so:

/api/v1/block/[hash]

Then as the api progresses, a v2 of the api can be released without breaking apps using v1

http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#versioning

**Comment by chrisarnesen** [comment]
Alternatively, the next version of the API could be /api2.

**Comment by maraoz** [comment]
@liferealized Agreed. Since creating the insight api we have learned a lot more about API good practices. We're planning to do a much better job with our current project https://github.com/bitpay/bitcore-node
